--- libmsi/string.c.orig	2020-05-05 18:16:08.000000000 +0200
+++ libmsi/string.c	2020-05-05 18:15:46.000000000 +0200
@@ -22,7 +22,13 @@
 
 #include <stdarg.h>
 #include <assert.h>
-#include <endian.h>
+#ifdef __APPLE__
+  #include <machine/endian.h>
+  #include <libkern/OSByteOrder.h>
+  #define le16toh(x) OSSwapLittleToHostInt16(x)
+#else
+  #include <endian.h>
+#endif
 #include <gsf/gsf-msole-utils.h>
 
 #include "debug.h"
